lib/bolt/cli.rb

```diff
@@ -831,13 +843,16 @@ def initialize_project
       # Puppetfile, install the specified modules. The module installer will
       # resolve dependencies, generate a Puppetfile, and install the modules.
       if options[:modules]
+        require 'bolt/module_installer'
```

Comment:
If we want to leave all these requires in, it'd be best to move them to after we validate the puppetfile doesn't exist so we don't load this then immediately error.